skiko/src/jvmMain/kotlin/org/jetbrains/skiko/SkikoProperties.kt

```diff
@@ -29,7 +29,21 @@ internal object SkikoProperties {
         when(text) {
             "SOFTWARE" -> return GraphicsApi.SOFTWARE
             "OPENGL" -> return GraphicsApi.OPENGL
-            else -> return GraphicsApi.OPENGL
+            "DIRECT3D" -> {
+                return if (hostOs == OS.Windows) GraphicsApi.DIRECT3D else bestRenderApiForCurrentOS()
```

Comment:
Guess we'd better throw an exception on else branch for clarity.

Reply:
Done.